plugins/bulk_upload/xml/batch/BulkUploadEngineXml.php

```diff
@@ -1024,14 +1024,17 @@ protected function sendItemAddData(KalturaBaseEntry $entry ,KalturaResource $res
 		KBatchBase::unimpersonate();
 
 		$nonCriticalErrors = '';
-		foreach($requestResults as $requestResult)
+		if($requestResults)
 		{
-			if (is_array($requestResult) && isset($requestResult['code']))
-				$nonCriticalErrors .= $requestResult['message']."\n";
-			if ($requestResult instanceof Exception)
-				$nonCriticalErrors .= $requestResult->getMessage()."\n";
+			foreach($requestResults as $requestResult)
+			{
+				if (is_array($requestResult) && isset($requestResult['code']))
+					$nonCriticalErrors .= $requestResult['message']."\n";
+				if ($requestResult instanceof Exception)
+					$nonCriticalErrors .= $requestResult->getMessage()."\n";
+			}
 		}
```

Comment:
Add a message to the log signifying we didn't get anything in the multirequest result